bio_hansel/main.py

```diff
@@ -4,18 +4,20 @@
 import argparse
 import logging
 import sys
-import os
-import re
 from typing import Optional, List, Any, Tuple
+
+import attr
+import os
```

Comment:
Isn't "os" a built-in module, shouldn't it be in the first group?

Reply:
Yup! Fixed in acf4335341097e0583a4be7c271fa860cc7312de